sshcommand

```diff
@@ -58,6 +58,13 @@ case "$1" in
 
     getent passwd "$USER" > /dev/null || false
     USERHOME=$(sh -c "echo ~$USER")
+
+    NEW_KEY=$(grep "NAME=\\\\\"$NAME"\\\\\" "$USERHOME/.ssh/authorized_keys" || true)
+    if [ ! -z "$NEW_KEY" ]; then
+      echo "Duplicated SSH Key name"
```

Comment:
We should start using this pattern and use this pattern everywhere if possible (though that can be a separate commit). Follow the unix way :)